internal/conf/conf.go

```diff
@@ -57,6 +63,7 @@ func Load(configPath string) (*Config, error) {
 	}
 
 	var config Config
+	defaults.Set(&config)
```

Comment:
I don't think it's worth using a package for setting defaults for two vars, let's do it manually after line 70.

Reply:
You're right, but maybe it's worth implementing to cover case with passing minimal yaml config in the future?

Reply:
Yeah, ideally, we should be able to embedded a minimal YAML config, then overwrite values from the provided YAML.

Reply:
I decided to create another PR for this one, right now i have assigned them in code.